backend/spec/requests/tags_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+RSpec.describe 'TagsController' do
+  describe 'POST /tags' do
+    context 'パラメーターが有効な場合' do
+      let(:valid_tag_params) { { label: Faker::Lorem.sentence(word_count: 5), color: :white, priority: 0 } }
+
+      it 'タグが追加され、204が返る' do
+        aggregate_failures do
+          expect do
+            post '/tags', params: valid_tag_params, as: :json
+          end.to change(Tag, :count).by(1)
+
+          expect(response).to have_http_status(:no_content)
+          expect(response.body).to be_empty
+        end
+      end
+    end
+
+    context 'パラメーターが無効な場合' do
+      let(:invalid_tag_params) { { label: '', color: :white, priority: 0 } }
```

Comment:
Nits

contextみたらわかるのと、
同じようなletがcontext内に存在しないため、prefixいらないと思います！

```suggestion
      let(:params) { { label: '', color: :white, priority: 0 } }
```